ember/config/environment.js

```diff
@@ -120,5 +130,12 @@ module.exports = function (environment) {
     // here you can enable a production-specific feature
   }
 
+  ENV["ember-simple-auth-oidc"].host = [
```

Comment:
Why don't you just write that config option directly into the config object? The variables `keycloakHost` and `keycloakRealm` are used nowhere else

Reply:
You are right. Moved it into the config object.